src/main/kotlin/no/nav/familie/ba/mottak/task/OpprettJournalføringOppgaveTask.kt

```diff
@@ -77,6 +77,10 @@ class OpprettJournalføringOppgaveTask(private val journalpostClient: Journalpos
         }
     }
 
+    private fun erFeatureDeaktivert(): Boolean {
```

Comment:
Hva er denne togglen? Burde vel ha litt mer beskrivende navn? Finner den ikke i unleash heller. Treger vi den da?

Reply:
En toggle som kan brukes til å skru av integrasjonen mot ba-sak og infotrygd. La det til etter problemene det skapte for Eivind da han skulle opprette opplæringsoppgaver i preprod, just in case. Men er vel tvilsomt om vi trenger den noe mer

Reply:
fjernet den, jeg